app/views/govuk_publishing_components/components/_autocomplete.html.erb

```diff
@@ -0,0 +1,42 @@
+<%
+add_gem_component_stylesheet("autocomplete")
+display_number_suggestions ||= 5
+suggestion_icon ||= ''
+request_debounce_time ||= ''
+source ||= nil
+position_results ||= nil
+submit_form_on_select ||= false
+idPostfix ||= "#{SecureRandom.hex(4)}"
+
+component_helper = GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(local_assigns)
+component_helper.add_class("gem-c-autocomplete")
+component_helper.add_data_attribute({ module: "gem-autocomplete" })
+component_helper.add_data_attribute({ source: source })
+component_helper.add_data_attribute({ id_postfix: idPostfix })
+
+component_helper.add_data_attribute({ display_number_suggestions: display_number_suggestions })
+component_helper.add_data_attribute({ suggestion_icon: suggestion_icon })
+component_helper.add_data_attribute({ submit_on_select: submit_form_on_select })
+
+if position_results.present?
+  component_helper.add_data_attribute({ position_results: 'absolute' })
```

Comment:
I haven't looked at the JS in detail yet but it looks like this data attribute is read by the JS then used to configure the autocomplete. Have you considered instead adding this as a modifier class and controlling the position of the results list entirely using CSS?

Reply:
good point, sorted i think